lms/static/js/discovery/main.js

```diff
@@ -0,0 +1,22 @@
+RequireJS.require([
+    'jquery',
+    'backbone',
+    'js/discovery/app',
+    'js/discovery/collection',
+    'js/discovery/form',
+    'js/discovery/result_list_view',
+    'js/discovery/filter_bar_view',
+    'js/discovery/search_facets_view'
+], function ($, Backbone, App, Collection, DiscoveryForm, ResultListView, FilterBarView, FacetsBarView) {
```

Comment:
Hmmm.... there's a few issues here that @wedaly mentions:
i) Use of RequireJS in LMS
- We were asked to start using RequireJS in LMS, it seems like adding the optimizer is really out of scope for what we are trying to do, but @dino-cikatic is experimenting to see if we can successfully integrate it.
ii) Cache busting mechanism - That's not something that we've explicitly done ourselves, seems like that is what is being done in general by LMS. Again, @dino-cikatic is investigating to see if he can compress by adding these to the assets pipeline (although I believe that there are some issues with RequireJS and pipeline processing that might scupper the process).

So, short summary is that we've got @dino-cikatic investigating solutions around this, but I'd like to get a temperature read from @smagoun, @wedaly and @pbaruah on how important immediately addressing this is - depending upon @dino-cikatic's investigation it might be better to address those issues in follow-on work?